test/highlight.spec.js

```diff
@@ -220,11 +220,11 @@ describe('highlight', () => {
 
   it('highlight sublanguages', done => {
     const str = '<node><?php echo "foo"; ?></node>';
-    const result = highlight(str, {lang: 'xml'});
+    const result = highlight(str, { autoDetect: true });
     result.should.eql([
-      '<figure class="highlight xml"><table><tr>',
+      '<figure class="highlight php-template"><table><tr>',
```

Comment:
`xml` may be detected as `php-template`, see upstream [changelog](https://github.com/highlightjs/highlight.js/blob/e8ca7fb48137b343591047277a9c7dbad43cd410/VERSION_10_BREAKING_CHANGES.md#version-10-breaking-changes).